.env

```
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_ADDRESS = "https://bootcamp-api.codeit.kr/api/"
```

address라는 표현보다는 base url 이라는 표현이 더 좋을 것 같네요~
```suggestion
NEXT_PUBLIC_API_BASE_URL = "https://bootcamp-api.codeit.kr/api/"
```